### ISSUE TITLE
JWT token validation is not working, I have added cat by passing wrong token.

### ISSUE THREAD

**Issue by kamleshgorasiya** [DavideViolante/Angular-Full-Stack#211]
I want to contribute to this project for this security, localization support and fixing vulnerable dependency support. 
I am taking the fork from this branch will submit a pull request.

**Comment by dougal83** [comment]
> Here again, we have one question mark, Is it good to store token in local storage?
Yes, it is designed to use local storage and stores by domain.  Was the 'other site' also on your local machine and use the same name for the local storage variable?  Ideally the code will handle the conflict you've mentioned by typically clearing local storage and treating you like an anonymous user without error.
